apps/sms/js/thread_list_ui.js

```diff
@@ -530,7 +490,64 @@ var ThreadListUI = {
         text: 'delete',
         className: 'danger'
       }
-    ).then(performDeletion.bind(this));
+    ).then(performDeletion.bind(this, selected));
+  },
+
+  deleteThreadDraftUI:
```

Comment:
call this function `deleteThreadDraft` and the other function `performDeleteThreadDraft`